README.md

```diff
@@ -1,8 +1,64 @@
 # NFT rank system for buying concert tickets
-### SUU project
+
+## SUU project
+
+The aim of this project is to provide a learning platform for Web3 technologies. 
 
 - frontend - simple react webpage for interacting with contracts
 - backend - truffle project, running local node with NFT tickets and ranks smart contracts
 
+Documentation is available [here](https://docs.google.com/document/d/1TLMwFXpCthf7yNgOx3ijIcUjzIAGXIpwliJWbig1XsA/edit).
+
+## Prerequisites
+
+- Docker
+- Node.js
+
+## Setting up
+
+1. Clone this repository and navigate to root directory.
+   ```
+   git clone https://github.com/war-in/suu-nft-project
+   ```
+
+1. Build and run Docker containers in a compose file:
```

Comment:
```suggestion
2. Build and run Docker containers in a compose file:
```

Reply:
Markdown autonumbers the ordered list when enumerated with `1.` (You can see on the branch).